database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,12 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UsersTableSeeder::class);
+
+        \Illuminate\Support\Facades\DB::statement('SET FOREIGN_KEY_CHECKS=0');
```

Comment:
To avoid of such hacks of "constraint violation" just reorder your seeders. I'm sure you'll verify why this error happens and how to fix it without that line ;)

Reply:
Fixed.